test/test-remote/looker/src/logs/dummystream.ts

```diff
@@ -218,7 +218,7 @@ export class LogSeries extends TimeseriesBase {
       this.nFragmentsConsumed + 1,
       this
     );
-    for (let i = 0; i < this.n_samples_per_series_fragment; i++) {
+    for (let i = 0; i < this.n_entries_per_stream_fragment; i++) {
```

Comment:
haha, I just did the rename into the other direction, because "sample" is more generic for both logs and metrics, whereas "entry" is logs-oriented. I did the rename from bottom to top, from within the time series classes -- but didn't follow through with the outest one, the CLI argument yet.

Reply:
reverting this in upcoming PR